client/lib/hawk.js

```diff
@@ -47,7 +47,7 @@ define(['sjcl'], function (sjcl) {
      */
 
     header: function (uri, method, options) {
-
+      /*eslint complexity: [2, 21] */
```

Comment:
Wat??

Actually, what are these _/client/lib/**_ files? Are these _our_ libraries, or kind of like third party scripts that we should add to .eslintignore?

/cc @zaach @vladikoff @shane-tomlinson